src/index.tsx

```diff
@@ -194,12 +196,12 @@ A11yDialog.propTypes = {
 
   // Object of classes for each HTML element of the dialog element.
   // See: https://a11y-dialog.netlify.app/usage/markup
-  classNames: PropTypes.exact({
-    container: PropTypes.string,
-    overlay: PropTypes.string,
-    dialog: PropTypes.string,
-    title: PropTypes.string,
-    closeButton: PropTypes.string,
+  classNames: PropTypes.shape({
+    container: PropTypes.string.isRequired,
+    overlay: PropTypes.string.isRequired,
+    dialog: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    closeButton: PropTypes.string.isRequired,
```

Comment:
🔍 This is kind of a weird one. We want `classNames` to be optional, and we don't require _all_ of these classes to be present, but when they _are_ present, they have to be strings. Hence: `exact` becomes `shape`, and each string `isRequired`